infra/modules/aws-host/main.tf

```diff
@@ -190,3 +190,23 @@ locals {
   all_instances = merge(local.api_instances, local.bulk_instances)
 }
 
+# script to test ALL connectors
+resource "local_file" "test_all_script" {
```

Comment:
exactly the same as gcp-host. Extract to own file and do sym link?

Reply:
Terraform doesn't seem to like to follow sym links. I tried it when splitting `main.tf`/`variables.tf` ... I have not tried within modules, only at top level (root) configurations.  But yeah, maybe could try more.

There is A LOT that's very, very similar between AWS and GCP cases.